doc/man3/ASN1_TIME_set.pod

```diff
@@ -179,6 +179,10 @@ starting with B<ASN1_UTCTIME> and B<ASN1_GENERALIZEDTIME> act only on that
 specific time format. The functions starting with B<ASN1_TIME> will operate on
 either format.
 
+Users familiar with RFC822 should note that even if specifying the flag
```

Comment:
I'd suggest changing _even if_ with _when_.  It reads better to me but I'm not going to be a stickler about it.

Reply:
So changed. You're much closer to the language than I am. And thanks for the approval.